virtual_rainforest/models/hydrology/model_schema.json

```diff
@@ -11,6 +11,12 @@
                     "exclusiveMinimum": 0,
                     "default": 0.5
                 },
+                "init_groundwater_saturation": {
+                    "description": "Initial ground water saturation for all layers",
```

Comment:
Am I right in thinking this is a ratio and therefore unitless?

Reply:
yes, that is correct. Should I add this to the description or is there a "unit" option?

Reply:
Up to you. You could mention that it's unitless, but the reason I asked was in case it was e.g. millimetres in which case it would def be worth mentioning.